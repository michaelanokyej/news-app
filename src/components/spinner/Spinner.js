import React from "react";
import "./Spinner.css";

const spinner = () => (
  <div className="spinner__div">
    <div className="lds-circle">
      <div></div>
    </div>
  </div>
);

export default spinner;
