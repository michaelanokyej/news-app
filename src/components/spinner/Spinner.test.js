import React from "react";
import Spinner from "./Spinner"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"
import renderer from 'react-test-renderer'

describe(`BackDrop component`, () => {
    it(`renders the BackDrop feature`, () => {
        const wrapper = shallow(<Spinner />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});