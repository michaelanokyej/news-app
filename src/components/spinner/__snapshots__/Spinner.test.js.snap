// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackDrop component renders the BackDrop feature 1`] = `
<div
  className="lds-ellipsis"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
